docs/blog/2022-03-08-zephyr-3-0-upgrade-prep.md

```diff
@@ -13,16 +13,16 @@ their repositories now.
 GitHub Actions needs to use our latest Docker image to ensure continued compatibility with the ZMK codebase on Zephyr 3.0 (and beyond). You should:
 
 - Open `.github/workflows/build.yml` in your editor/IDE
-- Change `zmkfirmware/zmk-build-arm:2.5` to `zmkfirmware/zmk-build-arm:stable` wherever it is found
+- Replace its content with the [updated template](https://github.com/zmkfirmware/unified-zmk-config-template/blob/main/.github/workflows/build.yml)
 
-Once the changes are committed and pushed, the build will run as expected.
+The new configuration template does not include the pipeline definition itself, but rather references a central configuration file. Thus, you will not have to update this file again in the future, as all updates to the central configuration will automatically be reflected in your new pipeline runs.
```

Comment:
I'd rather we announce this with an additional new blog post, since this one was already announced a while ago, and read by folks.

Reply:
Makes sense, I removed it.